regression/getCoverage.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Get a unique number to prevent collision of output files
+starttime=`date +%s`
```

Comment:
I have made this change, slightly enhanced it to create a unique directory using mktemp and then put all the files under that directory.